omniledger/service/contracts.go

```diff
@@ -35,6 +35,10 @@ var ContractConfigID = "config"
 // ContractDarcID denotes a darc-contract
 var ContractDarcID = "darc"
 
+// ContractValueID denotes a contract that can store and update
+// key values.
```

Comment:
Is this a toy? Is this a caprice de Linus? Is this a critical future feature? Or for testing?

Am a bit concerned we are adding a super generic thing with no structure. Who's allowed to use this?

Reply:
Allowed is everybody with `spawn:value` access in a darc. I use it in the java tests. And it might also be used for more generic data storage.